src/components/OcAppSideBar.vue

```diff
@@ -70,9 +70,6 @@ export default {
 ```vue
   <template>
     <oc-app-side-bar>
-        <template slot="action">
-          <oc-button class="uk-float-right oc-mr-s"><oc-icon name="close" /></oc-button>
```

Comment:
it's just a documentation example. Questions remains the same though 😁

Reply:
As far as I understand, these few lines just "re-implement" the default action, which is the close button. I stumbled upon this because I tested something on the close-button defined in `<template>`, but it did not render. I can't see any difference on small screens either way 🤔

Reply:
Ah then it's nice to have them deleted actually!